src/muya/lib/parser/marked/lexer.js

```diff
@@ -233,6 +233,7 @@ Lexer.prototype.token = function (src, top) {
       let next = false
       let prevNext = true
       let listItemIndices = []
+      let isTaskListItem = false
```

Comment:
`isTaskList` and please link to your explanation below.

Reply:
>`isTaskList`

Do you mean I need to  rename `isTaskListItem` to `isTaskList`?

Reply:
Exactly